asset/css/markbind.css

```diff
@@ -302,3 +302,46 @@ panel[minimized]:not([expanded])::before {
 li.footnote-item:target {
     background-color: #eee;
 }
+
+/* "Copy" code block button */
+pre {
+    position: relative;
+}
+
+.copy-btn {
+    background-color: #b4b4b9;
+    border-radius: 0.25rem;
+    color: #f8f8ff;
+    display: inline-block;
+    font-size: 75%;
+    line-height: 1;
+    padding: 0.25em 0.4em;
+    position: absolute;
+    right: 0.5em;
+    text-align: center;
+    top: 0.4em;
+    vertical-align: baseline;
```

Comment:
Is `vertical-align: baseline;` necessary? It seems to be the default value.

https://www.w3schools.com/cssref/pr_pos_vertical-align.asp

Reply:
Resolved 👍